{"name": "previous orders section", "status": "skipped", "statusDetails": {"message": "XFAIL NO WAY TO TEST THIS RIGHT NOW\n\nAssertionError: assert False", "trace": "driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"2ad8638f-e223-464a-b02d-d691b518b423\")>\n\n    @allure.epic('Test Personal Area')\n    @allure.id(39)\n    @allure.title(\"previous orders section\")\n    @allure.description('Previous orders section will correctly show previous orders')\n    @allure.severity(allure.severity_level.NORMAL)\n    @pytest.mark.xfail(condition=lambda: True, reason='NO WAY TO TEST THIS RIGHT NOW')\n    def test_previous_orders_section(driver):\n>       assert False\nE       assert False\n\nTest_personal_area.py:78: AssertionError"}, "description": "Previous orders section will correctly show previous orders", "start": 1680022674734, "stop": 1680022674735, "uuid": "e5543a73-6e9f-4957-bfc5-96b70585936f", "historyId": "513b048b251aa6655194b0c69924c078", "testCaseId": "0c8368f62c799861e5c9a355eb78da35", "fullName": "Test_personal_area#test_previous_orders_section", "labels": [{"name": "as_id", "value": 39}, {"name": "epic", "value": "Test Personal Area"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.xfail(condition=<function <lambda> at 0x000001A7718C7A60>, reason='NO WAY TO TEST THIS RIGHT NOW')"}, {"name": "suite", "value": "Test_personal_area"}, {"name": "host", "value": "TWEEJ5"}, {"name": "thread", "value": "18924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_personal_area"}]}